industrial_ci/src/tests/source_tests.sh

```diff
@@ -118,6 +118,14 @@ function ici_combine_cpp_reports {
   # Filter out test files
   lcov --remove coverage.info "*/test/*" \
        --output-file coverage.info | grep -ve "^removing"
+  # Filter out ignored packages
+  local -a ignored_pkgs
+  ici_parse_env_array ignored_pkgs CODE_COVERAGE_IGNORE_PACKAGES
+
+  for pkg in "${ignored_pkgs[@]}"; do
+      lcov --remove coverage.info "$(pwd)/src/$TARGET_REPO_NAME/$pkg/*" \
+           --output-file coverage.info | grep -ve "^removing"
+  done
```

Comment:
Combine this with ignoring `*/test/*`, for those that is a relative path (doesn't start with wildcard `*`), full path will be added

Reply:
Why do you want to combine this with ignoring `*/test/*? If I want to exlcude a whole package `src/<repo_name>/pkg1` should be fine?